ansible/MANUAL_STEPS.md

```diff
@@ -108,8 +108,13 @@ Xcode Command-line tools are not enough to perform a full notarization cycle, fu
 As root:
 
 * Download Xcode: https://developer.apple.com/download/more/ - find non-beta version, open Developer Tools in browser, Networking tab, start download (then cancel), in Networking tab "Copy as cURL" (available in Chrome & FF)
+  * On OSX 11 we currently install 13.2.1
 * Download onto release machine using the copied curl command (may need `-o xcode.xip` appended to curl command) to `/tmp`
+  * If you have trouble on the command line, pasting into a shell script file can make pasting/editing to add -o xcode.xip easier
 * Extract: `xip --expand xcode.xip`
+  * This takes a long time since xcode.xip is 7-10G in size depending on version
+  * If you run out of space you can delete `/Users/build/workspace/*` to free up some space. Also make sure that
```

Comment:
FWIW, at least for https://github.com/nodejs/build/issues/3375 most of the reclaimed extra space we needed was from clearing the ccache by running
```console
sudo -s su - iojs
ccache --clear
```
although we did also reclaim a smaller amount of space by removing the workspace.